File: android/app/src/main/java/net/ipv64/kivop/dtos/PosterServiceDTOs/HangPosterPositionResponseDTO.kt
package net.ipv64.kivop.dtos.PosterServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class HangPosterPositionResponseDTO (
    var posterPosition : UUID,
    var postedAt : LocalDateTime,
    var postedBy : UUID,
    var image : String,
)

File: DTOs/Sources/PosterServiceDTOs/HangPosterPositionResponseDTO.swift
import Foundation

public struct HangPosterPositionResponseDTO: Codable {
    public var posterPosition: UUID
    public var postedAt: Date
    public var postedBy: UUID
    public var latitude: Double?
    public var longitude: Double?

    public init(posterPosition: UUID, postedAt: Date, postedBy: UUID, latitude: Double? = nil, longitude: Double? = nil) {
        self.posterPosition = posterPosition
        self.postedAt = postedAt
        self.postedBy = postedBy
        self.longitude = longitude
        self.latitude = latitude
    }
}
